server-spi-private/src/main/java/org/keycloak/authorization/store/PolicyStore.java

```diff
@@ -119,7 +119,11 @@ default List<Policy> findByResource(ResourceServer resourceServer, Resource reso
      * @param resource the resource. Cannot be {@code null}.
      * @param consumer consumer of policies resulted from the search
      */
-    void findByResource(ResourceServer resourceServer, Resource resource, Consumer<Policy> consumer);
+    default void findByResource(ResourceServer resourceServer, Resource resource, Consumer<Policy> consumer) {
```

Comment:
This is a `server-spi-private` API, so technically we are not required to add any default methods to simplify the transition. So you could remove it. If you want keep it, it might be missing a depreciation annotation? Same below for the other method.

Reply:
We no longer have those default methods.